.idea/dictionaries/ktsukanov.xml

```diff
@@ -27,6 +27,7 @@
       <w>libbz</w>
       <w>liblzma</w>
       <w>licensable</w>
+      <w>medgen</w>
```

Comment:
Indeed! And if I understand the workings of PyCharm correctly, even though the file is named after my username, it's only to avoid merge conflicts—the spellchecking rules (or rather, exceptions) should apply to everyone cloning this project.